xml/System.Net.Security/SslClientAuthenticationOptions.xml

```diff
@@ -99,7 +108,7 @@
         <ReturnType>System.Collections.Generic.List&lt;System.Net.Security.SslApplicationProtocol&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets a list of <see cref="T:System.Net.Security.SslApplicationProtocol" />.</summary>
```

Comment:
It may be obvious but thesis bit mask or protocols client is willing to negotiate. It is up to server to decide.

Since this is weird, it may be worth of mentioning that  `None` means OS default. (and it is currently recommended value e.g. more then just a default)

Reply:
>It may be obvious...
I added this comment to #4367 PR as there is similar comment for the SslProtocols so we can uniform the statements in one PR

>Since this is weird,
I extended the value section